tablecodec/tablecodec.go

```diff
@@ -599,47 +601,116 @@ func CutIndexKeyNew(key kv.Key, length int) (values [][]byte, b []byte, err erro
 	return
 }
 
-// PrimaryKeyStatus is the primary key column status.
-type PrimaryKeyStatus int
+// cutIndexValue cuts encoded index value into bytes slices.
+// This function is used only when NewCollationEnabled is true.
+func cutIndexValue(value []byte, length int) (values [][]byte, err error) {
+	// ignore tailLen
+	b := value[1:]
```

Comment:
What is a `tailLen`? Is there any document about it?

Reply:
tailLen mean the len of handle id(8 byte) and untouch(1 byte). So it can be 0, 1, 8 or 9.